### ISSUE TITLE
Stuck on "Opening https:// ..."

### ISSUE THREAD

**Issue by eddieloeffen** [Szpadel/chrome-headless-render-pdf#32]
When I run this command:

`chrome-headless-render-pdf --chrome-binary="C:\Program Files (x86)\Google\Chrome\Application\chrome.exe" --url=https://www.chromestatus.com/features --pdf=output.pdf`

I get up to "Opening https://www.chromestatus.com/features" and nothing else happens?

Full cmd log

```
C:\bin\urls to pdf> chrome-headless-render-pdf --chrome-binary="C:\Program Files (x86)\Google\Chrome\Application\chrome.exe" --url=https://www.chromestatus.com/features --pdf=output.pdf
Using C:\Program Files (x86)\Google\Chrome\Application\chrome.exe
Waiting for chrome to became available
Connected!
Connected to HeadlessChrome/65.0.3325.181, protocol 1.2
Opening https://www.chromestatus.com/features
```

Chrome Headless opens up and does not close. All of these chrome processes are spawned when printing, and I started with no other chrome.exe open.

![image](https://user-images.githubusercontent.com/2539705/39023552-8028f74c-448f-11e8-8780-b648e06da727.png)

**Comment by Szpadel** [comment]
this looks like #29,
is it present also in 1.7.0?

**Comment by Szpadel** [comment]
Ok, I see that this is different issue,
But I'm not sure what can I do about it, this page isn't loading in chrome headless at all.
I'm not sure what is causing that

**Comment by Szpadel** [comment]
I think I found small regression for some pages, should now be fixed in 1.7.1
But site you mentioned isn't working correctly on chrome headless, and i currently don't know why exactly

**Comment by eddieloeffen** [comment]
Hmm ok, thanks for looking in to this anyway.
